docs/ATOM.md

```diff
@@ -0,0 +1,139 @@
+# ATOM
+
+Irisnet-crypto has completed the docking cosmos main network(chain-id:cosmoshub-2),supports signatures for the following transactions
+
+- Transfer
+- Delegate
+- Unbond
+- Redelegate
+- SetWithdrawAddress
+- WithdrawDelegationReward
+- WithdrawValidatorCommission
+
+Construct a valid transaction that requires user to construct the following transaction request:
+
+```json
+{
+  "chain_id": "",
+  "from": "",
+  "account_number": 0,
+  "sequence": 0,
+  "fees": {"denom": "uatom", "amount": 100},
+  "gas": 10000,
+  "memo": "",
+  "type": "",
+  "msg": msg
+}
+```
+The parameters are as follows:
+- chain_id : blockchain's chain_id
+- from : Signer's address of the transaction
+- account_number : you can get it from lcd's 'auth/accounts/ api'
+- sequence : you can get it from lcd's 'auth/accounts/ api'
+- fees : transaction fee
+- gas : gas limit
+- memo : transaction description(Up to 100 characters)
+- type : transaction type,enumeration values are as follows
+    - transfer
+    - delegate
+    - undelegate
+    - begin_redelegate
+    - set_withdraw_address
+    - withdraw_delegation_reward
+    - withdraw_validator_commission
+- msg : the specific transaction content, different transactions, and data structure are also different. For details, please refer to the following example.
```

Comment:
different transactions, and data structure are also different. 
->
different type of transaction has different structure